### PR TITLE
Feat/response in use indicator

### PR DESCRIPTION
**Pull request by kenrilea**
https://app.asana.com/0/1170145499389908/1169689593066905

- feat: indicator in the story editor when a response is used in another story
- chore: re-usable HoverablePopup component (semantic Uis is buggy)

### CODE REVIEW

File: botfront/imports/ui/components/stories/common/BotResponsesContainer.jsx [hunk 1]
```diff
@@ -154,6 +219,10 @@ BotResponsesContainer.propTypes = {
     onDeleteAllResponses: PropTypes.func,
     enableEditPopup: PropTypes.bool,
     tag: PropTypes.string,
+    // projectId: PropTypes.string.isRequired,
```
Comment (GuillaumeTech) [inline botfront/imports/ui/components/stories/common/BotResponsesContainer.jsx:222]: If it's commented, remove it ?

File: botfront/imports/ui/components/stories/common/style.import.less [hunk 2]
```diff
@@ -167,23 +167,40 @@
...
-        color: #767676;
-        font-size: 0.78571429rem;
-        opacity: 0;
-        transition: opacity 0.1s ease-in-out;
-        padding: 0 5px;
-        text-overflow: ellipsis;
-        overflow: hidden;
-        text-align: right;
+        .response-name {
+            // max-width: 22%;
```
Comment (GuillaumeTech) [inline botfront/imports/ui/components/stories/common/style.import.less:178]: Remove it ?

File: botfront/imports/ui/components/stories/common/nlu_editor/NluModalContent.jsx [hunk 3]
```diff
@@ -320,6 +320,7 @@ const NLUModalContent = (props) => {
                             )}
                             content={(
                                 <ConfirmPopup
+                                    title=''
```
Comment (GuillaumeTech) [inline botfront/imports/ui/components/stories/common/nlu_editor/NluModalContent.jsx:323]: Why is it needed to add an empty title ?

**Comment by GuillaumeTech** [comment]
I missed a minor thing the "utter_..."  is not bold as in the task description

**Comment by znat** [comment]
The app crashes when clicking the links in the popup. See video in Asana

**Pull request merged**